public/style.css

```diff
@@ -1,5 +1,16 @@
 @import "global.css";
 
+::placeholder{
+  color: var(--light-text);
```

Comment:
This should be in the global.css

Reply:
I think we're reserving globals for the variables. Right now all the css is global since every page is using it.